examples/training/semantic_segmentation/pascal_voc/deeplab_v3.py

```diff
@@ -56,49 +55,80 @@
 global_batch = local_batch * strategy.num_replicas_in_sync
 base_lr = 0.007 * global_batch / 16
 
+# TODO(tanzhenyu): add a diff dataset.
+# all_ds = load(split="sbd_train", data_dir=None)
+# all_ds = all_ds.concatenate(load(split="sbd_eval", data_dir=None))
+# train_ds = all_ds.take(10000)
+# eval_ds = all_ds.skip(10000).concatenate(load(split="diff", data_dir=None))
 train_ds = load(split="sbd_train", data_dir=None)
 eval_ds = load(split="sbd_eval", data_dir=None)
 
-resize_layer = tf.keras.layers.Resizing(512, 512)
+resize_layer = tf.keras.layers.Resizing(512, 512, interpolation="nearest")
+
+image_size = [512, 512, 3]
+
+
+def flip_fn(image, cls_seg):
```

Comment:
our RandomFlip KPL supports seg masks -- can we use that here instead?

Reply:
added a TODO, will test the BEFORE and AFTER difference.